actioncable/lib/action_cable/channel/streams_handlers/custom.rb

```diff
@@ -0,0 +1,29 @@
+module ActionCable
+  module Channel
+    module StreamsHandlers
+      class NullCoder
+        class << self
+          def decode(val)
+            val
+          end
+
+          alias encode decode
+        end
+      end
```

Comment:
This is a no-op coder, hence the existing "identity" language. Introducing "null" language ambiguously suggests that it's coding as null/nil or doing null coding.